src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/Protocol.php

```diff
@@ -77,4 +77,9 @@ public function asArray(): array
             'type' => $this->getProtocol(),
         ];
     }
+
+    public function getManagedProtocol(): string
```

Comment:
Please cover this method in test. Simple as it is.. It might prove useful at a later stage.

Reply:
Added a few protocol tests